app/src/pages/project/SpanFilterConditionField.tsx

```diff
@@ -298,6 +313,7 @@ export function SpanFilterConditionField(props: SpanFilterConditionFieldProps) {
         </AddonBefore>
         <CodeMirror
           css={codeMirrorCSS}
+          indentWithTab={false}
```

Comment:
was from other testing but i don't think we want this anyways, tab now moves to next focusable element on page, as opposed to tab the query over

Reply:
Oh good catch. Wish tab was auto-complete